src/main/java/com/minecolonies/coremod/entity/ai/minimal/EntityAIEatTask.java

```diff
@@ -141,6 +142,11 @@ public boolean shouldExecute()
             return true;
         }
 
+        if (citizen.getCitizenDiseaseHandler().isSick() && citizen.getCitizenSleepHandler().isAsleep())
```

Comment:
Why should citizen stop eating when they are sick?
Sick people consume food and resources from the colony.

Reply:
Indeed they should still consume resources, atm we have the eating AI below the sick AI so atm they arent eating at all. I've increased the eating AI priority so they eat again. This change is only to not eat during treating, as citizens are asleep and sick(happens when the hospital treats them). Normally the sleep AI has prio over the eat AI, just not in this case where the sickness AI makes them "sleep" in the hospital bed during treatment.

Reply:
Ah okey.
That sounds somewhat logical.
Approced then.